### DISCUSSION TITLE
Discussions feature is quite nice

### DISCUSSION THREAD

**Discussion by AtreyeeS** [registerrier/gammapy#12] [General]
I like this! This is much more informal than filing issues, and gives the flexibility to initiate several discussions at once.
What do you say @registerrier ? and why can't I tag Luca?

**Comment by luca-giunti** [comment]
I also find it nice. However I have the impression that it is prone to become a little messy @AtreyeeS I can tag you ;)

**Reply by AtreyeeS** [reply]
And now we can play tag here @luca-giunti 😁

**Reply by AtreyeeS** [reply]
I agree about being messy, but at least we can use the search box, so its better than slack...

**Comment by AtreyeeS** [comment]
Maybe we can have a dev-discussions category as well?

**Comment by registerrier** [comment]
Technically, I think it is possible to refer to specific discussions with the usual hashtag comment see e.g. #10 . 
But I wonder if we can also link to real issues, e.g.  GH-3000  or #3001

It is apparently possible to transform an issue in a discussion.

**Reply by luca-giunti** [reply]
This platform offers a lot of flexibility, but it will need to be maintained by someone to make sure issues/discussion/Q&A/ideas/whatever are in the right place, and correctly cross-linked. Isn't it preferable in this sense to have a simpler interface just having e.g. Q&A?

**Reply by registerrier** [reply]
We should start simple indeed. Maybe Q&A, announcements/general for communications, possibly issue/feature request.
I am less convinced by the show and tell, but who knows?

Of course we need to have someone(s) in charge of following on discussions, adding labels, possibly pinging the people most likely to know how to answer, or to block problematic discussions.
